department-of-veterans-affairs/caseflow
# APPEALS-35488: Update UAT Seed Scripts for Correspondence

Resolves [APPEALS-35488](https://jira.devops.va.gov/browse/APPEALS-35488)

# Description
Updated seed Correspondence Seed Scripts to be able to run in a Rake task, so that we can create seed data in UAT for internal/external UAT. 

## Acceptance Criteria

## Testing Plan
1. Open up a Rails Console and run these commands:
2. `RequestStore[:current_user] = User.find_by(css_id: 'BVADWISE')`
3. `MailTeam.singleton.add_user(RequestStore[:current_user])`
4. `file_number = Veteran.first.file_number`
5. `Rake::Task['correspondence:create_correspondence_test_data'].invoke`
6. At the command enter the file number from step 4.
7. When the script finishes, log into Caseflow as BVADWISE.
8. Navigate to Queue and click the Switch Views dropdown. Select **Your Correspondence** from the dropdown.
9. Check each tab within the table for generated seed data.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
